e2e/features/address-manual.feature

```diff
@@ -0,0 +1,38 @@
+Feature: Address details are entered manually without using postcode lookup
+
+  Background:
+    Given I am signed in as a Legal Rep
+    And I create a new case
+    And I complete the Home Office reference page
+    And I complete the Basic details page
+
+  @create-case @lodge-appeal@alternate @RIA-691
+  Scenario: Lodge an appeal application with a manually entered address
+
+    Given I am signed in as a Legal Rep
+    Given I create a new case
+    Given I complete the Home Office reference page
+    Given I complete the Basic details page
+
+    Given I am on the Your client's address page
+    When I select Yes for the Does the appellant have a fixed address? field
+    And I click the I can't enter a UK postcode link
+    And I type 11 Downing Street for the Building and Street field
+    And I type London for the Town or City field
+    And I type SW1A 2AB for the Postcode/Zipcode field
+    And I type United Kingdom for the Country field
+    And I click the Continue button
+
+    Given I complete the Why is your client appealing? page
+    Given I complete the New matters page
+    Given I complete the Has your client appealed against any other UK immigration decisions? page
+    Given I complete the Your own reference number page
+    When I click the Save and continue button
+    Then I should see an alert confirming the case has been created
+
+    Given I click the Case details tab
```

Comment:
This would probably make more sense as a `When`

Reply:
👍